module/VuFind/src/VuFindTest/Feature/ConfigPluginManagerTrait.php

```diff
@@ -112,4 +115,46 @@ protected function getMockFailingConfigPluginManager(
             ->will($this->throwException($exception));
         return $manager;
     }
+
+    /**
+     * Add config plugin manager and required services to a mock container
+     *
+     * @param \VuFindTest\Container\MockContainer $container Mock Container
+     * @param array                               $config    Module config
+     *
+     * @return void
+     */
+    protected function addConfigPluginManagerToContainer(
+        \VuFindTest\Container\MockContainer $container,
+        array $config
+    ): void {
+        $this->addPathResolverToContainer($container);
+        $configHandlerPluginManager = new \VuFind\Config\Handler\PluginManager(
+            $container,
+            $config['vufind']['config_reader']
```

Comment:
Is this using the wrong config section?

```suggestion
            $config['vufind']['config_handler']
```

Reply:
Good catch!

Reply:
It actually has to be `$config['vufind']['plugin_managers']['config_handler']`. Fixed it.